tests/integration/kubernetes/k8s-openvpn.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2025 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+    # Getting following error for nydus scenarios for the init container (which is a runc pod!)
+    # Cannot pull alpine image, error unpacking image: failed to extract layer sha256:<XXX>: failed to get reader from content store: content digest sha256:<XXX>: not found
+    [ "${SNAPSHOTTER:-}" == "nydus" ] && skip "openvpn tests not supported with nydus snapshotter"
```

Comment:
Two jobs are currently failing. The init runc pod gets stuck creating the DH key:
`# Generating DH parameters, 2048 bit long safe prime`:
- https://github.com/kata-containers/kata-containers/actions/runs/17805462480/job/50618988481?pr=11818
- https://github.com/kata-containers/kata-containers/actions/runs/17805462480/job/50618988472?pr=11818
  (- both are target_arch `s390x`)